tasty-plutus/src/Test/Tasty/Plutus/TestData.hs

```diff
@@ -180,6 +182,7 @@ fromArbitraryMinting ::
   , Show redeemer
   , Arbitrary redeemer
   ) =>
-  (redeemer -> Example) ->
+  (redeemer -> Value -> Example) ->
```

Comment:
This also requires an `@since` bump, a major version raise, and a CHANGELOG entry, as it changes the public API.